currency-converter/src/components/Dropdown.tsx

```diff
@@ -1,8 +1,14 @@
-import Select, { StylesConfig } from 'react-select';
+import Select, { OnChangeValue, StylesConfig } from 'react-select';
 
+export interface IDropdownOption {
```

Comment:
Now that this interface is being used in more than one files, we can move it to a generic file, i.e., types.ts